kubernetes/config-maps/concent-api/local_settings.py.j2

```diff
@@ -46,6 +47,10 @@ CELERY_BROKER_URL = 'amqp://rabbitmq.default.svc.cluster.local:5672'
 
 EMAIL_SUBJECT_PREFIX = "[{{ gke.cluster }}:concent-api] "
 
+PAYMENT_BACKEND = 'core.payments.sci_backend'
+GETH_CONTAINER_ADDRESS = 'http://geth.default.svc.cluster.local:8545'
+CONCENT_ETHEREUM_ADDRESS = '0xa5f3cd35385755e64b47f20a5d7334d7013f8015'
```

Comment:
Oh, I realized just now that this key should really be in `vars.yml`.

Reply:
Done.